.changeset/twelve-baboons-obey.md

```diff
@@ -0,0 +1,20 @@
+---
+'astro': minor
+---
+
+Astro integrations now accept a logger. The logger is available to all hooks as 
+an additional parameter:
```

Comment:
Quick review, but commented on #7791 that it might be preferable to include `logger` in the first argument if we can.

```suggestion
Integrations can now log messages using Astro’s built-in logger.
The logger is available to all hooks as an additional parameter:
```